src/test/java/de/retest/ui/descriptors/RootElementTest.java

```diff
@@ -32,8 +32,8 @@
 
 	@Test
 	public void applyChanges_adds_inserted_components() throws Exception {
-		final Element element = new Element( "wesdf", compIdentAttributes, new Attributes(),
-				Collections.singletonList( new Element( "asdasg", childIdentAttributes0, new Attributes() ) ) );
+		final Element element = Element.withChildren( "wesdf", compIdentAttributes, new Attributes(), Collections
+				.singletonList( Element.withoutChildren( "asdasg", childIdentAttributes0, new Attributes() ) ) );
```

Comment:
Can't we use the varargs ctor and just pass `Element.withoutChildren( ... )` without wrapping it into `singletonList`?